e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -1076,6 +1077,12 @@ describe("scenarios > search", () => {
 
 describeWithSnowplow("scenarios > search", () => {
   const PAGE_VIEW_EVENT = 1;
+  const NEW_SEARCH_QUERY_EVENT = 1;
```

Comment:
Just put in some variables so I could think about the events a little more clearly. Let me know if I should revert to numbers

Reply:
Please use `expectGoodSnowplowEvent`, ignore the helper that checks the number.

Reply:
Yep I agree, I really didn't like writing this